src/components/editor/SinopiaResourceTemplates.jsx

```diff
@@ -73,14 +73,9 @@ class SinopiaResourceTemplates extends Component {
   }
 
   groupDataPromise = (groupName, name) => new Promise(async (resolve) => {
-    // await resolve(instance.getResourceWithHttpInfo(groupName, name, { acceptEncoding: 'application/json' }))
     await resolve(getResourceTemplate(name, groupName))
   })
 
-  resourceToName = (resource) => {
```

Comment:
Not a fan of removing this method per the above comment. Also don't love that we have this same method defined in multiple components. Combined with my above related comment, what this says to me is that we ought to have `resourceToName` implemented elsewhere (in some utilities class?) and that both the `SinopiaResourceTemplates` and `UpdateResourceModal` components should import that function. **And** I'm fully on board with replacing the implementation so that it uses a regex instead of substrings.

Reply:
see comments above

Reply:
:man_dancing: